src/requirements.txt

```diff
@@ -0,0 +1,2 @@
+boto3  # AWS SDK for Python
+jsonschema # Validate JSON settings schema
```

Comment:
I'm afraid there's no such package available in the default Lambda runtime: https://gist.github.com/gene1wood/4a052f39490fae00e0c3

We can either use something mentioned in the gist above, or use Docker to prepare a layer with all packages from requirements.txt

Reply:
I agree. 
We have just one use-case where we use configs validation - inside CDK code, before deployment we want to make sure
a) configs are present (where we expect to find them :) )
b) configs are valid.
So it tells us, we can deploy this version of configs to S3 and safely use it.

Manually changing configs directly on S3 is not good scenario for too many reasons :) 
So, deploy configs only via CDK process

Reply:
@NastassiaPaskrobka , may I ask you then to extract settings validation functionality into a separate module so that we would use it directly in CDK only? In that way Lambdas will use only settings readers, and not settings validation, and won't have jsonschema import issues.

Reply:
Sure, makes sense, I'll move it to infra_tooling_stack then

Reply:
Just realized that Validator uses settings reader also.
So once again open question where to put settings reader to be commonly accessed by lambdas and CDK :)